src/JSON/ZeroCopy/Deserializer.dfy

```diff
@@ -259,9 +278,19 @@ module {:options "-functionSyntax:4"} JSON.ZeroCopy.Deserializer {
       elems'
     }
 
+    lemma AboutTryStructural(cs: FreshCursor)
+      ensures
+        var sp := Core.TryStructural(cs);
+        var s0 := sp.t.t.Peek();
+        && ((!cs.BOF? || !cs.EOF?) && (s0 == SEPARATOR as opt_byte) ==> (var sp: Split<Structural<jcomma>> := sp; sp.cs.StrictSuffixOf?(cs)))
+        && ((s0 == SEPARATOR as opt_byte) ==> var sp: Split<Structural<jcomma>> := sp; sp.SplitFrom?(cs, st => Spec.Structural(st, SpecView)))
+        && ((!cs.BOF? || !cs.EOF?) && (s0 == CLOSE as opt_byte) ==> (var sp: Split<Structural<jclose>> := sp; sp.cs.StrictSuffixOf?(cs)))
+        && ((s0 == CLOSE as opt_byte) ==> var sp: Split<Structural<jclose>> := sp; sp.SplitFrom?(cs, st => Spec.Structural(st, SpecView)))
+    {}
+
     // The implementation and proof of this function is more painful than
     // expected due to the tail recursion.
-    function {:opaque} {:tailrecursion} Elements(
+    function {:timeLimit 30} {:vcs_split_on_every_assert} {:opaque} {:tailrecursion} Elements(
```

Comment:
```suggestion
    function {:vcs_split_on_every_assert} {:opaque} {:tailrecursion} Elements(
```

(And likewise below)

I assume this was here for debugging earlier. We should make sure we only include very generous wall-clock timeouts to maximize build reproducibility, and I don't see a huge value in bounding this more tightly (we could set an `:rlimit` instead if that's the intention)

Reply:
You are right, it's been used for debugging. I'll remove it everywhere.